csaf_2.1/json_schema/WHAT_IS_NEW_IN_CSAF_2_1.md

```diff
@@ -0,0 +1,161 @@
+    
+- **Date Field for Vulnerability Disclosure:**
+  - **CSAF 2.1:**  
+    - Uses `"disclosure_date"` to indicate when the vulnerability was disclosed to the public.  
+    - Also includes `"discovery_date"` to indicate when the vulnerability was initially discovered.
+    
+  - **CSAF 2.0:**  
+    - Uses `"release_date"` (for when the vulnerability was released into the wild) instead of `"disclosure_date"`.
+    
+- **Metrics vs. Scores:**
+  - **CSAF 2.1:**  
+    - Replaces the `"scores"` property with `"metrics"`.  
+    - Each metric item now contains a `"content"` object that can reference multiple scoring systems including:
+      - CVSS v2 and v3 (with support for both CVSS v3.0 and v3.1)
+      - **Newly added:** CVSS v4 and SSVC v1  
+    - The metric item also includes `"products"` and an optional `"source"`.
+    
+  - **CSAF 2.0:**  
+    - Used a `"scores"` array where each score item primarily referenced `"cvss_v2"` and `"cvss_v3"` alongside the list of `"products"`.
+    
+- **Remediation Enhancements:**
+  - **CSAF 2.1:**  
+    - In the `"remediations"` property, the `"category"` enumeration has been expanded to include new values:
+      - `"fix_planned"`
+      - `"optional_patch"`
+    - The other values such as `"mitigation"`, `"no_fix_planned"`, `"none_available"`, and `"vendor_fix"` remain.
+    
+  - **CSAF 2.0:**  
+    - The enumeration for remediation categories was limited to `"mitigation"`, `"no_fix_planned"`, `"none_available"`, `"vendor_fix"`, and `"workaround"`.
+
+----
+
+## Provider Metadata JSON Schema
+
+Below is a comparison between the CSAF provider metadata schemas for version CSAF 2.1 and CSAF 2.0.
+
+### Schema Identity and Required Properties
+
+Of course, updating the new 2.1 version references.
+
+- **Schema Declaration:**
+  - **CSAF Provider 2.1:**  
+    - Declares a `$schema` property and uses an `$id` pointing to the v2.1 URL.  
+    - The required list explicitly includes `"$schema"`.
+  
+  - **CSAF Provider 2.0:**  
+    - Uses an `$id` pointing to the v2.0 URL, but the `$schema` property is not required.
+    - The property `"metadata_version"` is fixed to `"2.1"`.
+
+
+
+
+## Distributions Structure
+
+- **Directory-Based Distribution:**
+  - **CSAF Provider 2.1:**  
+    - The distribution mechanism for directory-based distribution is encapsulated as an object under the property `"directory"`.  
+    - This object requires:
+      - `"tlp_label"` – its value is obtained via a reference to the CSAF 2.1 document’s TLP label definition (which, in CSAF 2.1, typically uses values such as `"AMBER"`, `"AMBER+STRICT"`, `"CLEAR"`, `"GREEN"`, and `"RED"`).  
+      - `"url"` – the base URL for the directory.
+
+  - **CSAF Provider 2.0:**  
+    • Uses a simpler approach where the directory distribution is provided as a single property named `"directory_url"` that directly holds the URL.
```

Comment:
Is the `•` intentional?

Reply:
Maybe someone is using word or open office :-) - I dashified it, totally.